### ISSUE TITLE
Cannot navigate to "Small media player" in the bottom with screen reader or keyboard

### ISSUE THREAD

**Issue by lohnn** [lohnn/lohnn_podcast#62]
The title says it all. It is not possible to set screen reader focus on the media player at the bottom.
